app/helpers/view_helper.rb

```diff
@@ -122,6 +122,17 @@ def time_today_or_tomorrow(time)
     end
   end
 
+  def date_and_time_today_or_tomorrow(time)
```

Comment:
From the prototype, we will be moving towards this over the above `time_today_or_tomorrow` method in other places too